me.id.webverify/webverifylib/src/main/java/me/id/webverifylib/IDmeCommonScope.java

```diff
@@ -0,0 +1,28 @@
+package me.id.webverifylib;
+
+import android.support.annotation.NonNull;
+
+/**
+ * The type of group verification.
+ */
+public enum IDmeCommonScope implements IDmeScope {
+  MILITARY("military"),
```

Comment:
yes, but this isn't a stable branch, maybe we can add test in another pull because I'm short of time. makes sense?